### PR TITLE
[KOGITO-7481] - Update ose-kube-rbac-proxy to v4.10.0

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

**Comment by spolti** [comment]
@radtriste I've update the kube-rbac-proxy to the latest version as well.

**Comment by radtriste** [comment]
looks like infinispan operaotr version is too old ... gonna check that

File: bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ metadata:
           "apiVersion": "rhpam.kiegroup.org/v1",
           "kind": "KogitoBuild",
           "metadata": {
-            "name": "ruleunit-quarkus-example"
+            "name": "process-quarkus-example"
```
Comment (radtriste) [inline bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:11]: are you sure about that change ?

Comment (spolti) [reply bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:11]: yes, process are not yet supported, we did this same change on 1.13.
However, I didn't update it manually, just run the command to generated the bundle.

**Comment by radtriste** [comment]
could not reproduce locally on another cluster. let's try again

**Comment by radtriste** [comment]
I think the problem is with the cluster. we can merge that one and I will check the used cluster

**Pull request merged**